help/Makefile.am

```diff
@@ -1,16 +1,16 @@
-include $(top_srcdir)/gnome-doc-utils.make
-dist-hook: doc-dist-hook
+@YELP_HELP_RULES@
 
-DOC_ID = hamster
-DOC_INCLUDES = legal.xml
-DOC_LINGUAS = cs da de el es fa fr gl hu pl pt_BR ro ru sl te zh_CN zh_HK zh_TW
+HELP_ID = hamster
 
-DOC_PAGES = \
+HELP_FILES = \
```

Comment:
Why rename all this stuff from `DOC_` to `HELP_`, is this because the Yelp makefile rules needs that?

Reply:
Yes, these are the variables used by [yelp-tools](https://wiki.gnome.org/Apps/Yelp/Tools/yelp.m4). It's not really needed until autotools are not used, it's just for compatibility.

Reply:
Ok, thanks for clarifying.